### DISCUSSION TITLE
can egui be used on linux framebuffer or gdiplus on Windows (without GPU)?

### DISCUSSION THREAD

**Discussion by walkthetalk** [emilk/egui#3263] [Q&A]
can egui be used on linux framebuffer or gdiplus on Windows (without GPU)?

**Comment by stonerfish** [comment]
egui is dependent of a gl system.
But a GPU is not needed for a gl driver.  Look into mesa for software gl rendering.

https://docs.mesa3d.org/drivers/llvmpipe.html
